client/me/notification-settings/comment-settings/index.jsx

```diff
@@ -16,50 +16,41 @@ import Navigation from '../navigation';
 import Card from 'components/card';
 import FormSectionHeading from 'components/forms/form-section-heading';
 import SettingsForm from 'me/notification-settings/settings-form';
+import QueryUserDevices from 'components/data/query-user-devices';
 import store from 'lib/notification-settings-store';
 import { fetchSettings, toggle, saveSettings } from 'lib/notification-settings-store/actions';
 import { successNotice, errorNotice } from 'state/notices/actions';
 
-const NotificationCommentsSettings = React.createClass( {
-	displayName: 'NotificationCommentsSettings',
-
-	mixins: [ observe( 'devices' ) ],
-
-	getInitialState() {
-		return {
-			settings: null
-		};
-	},
+class NotificationCommentsSettings extends Component {
+	state = { settings: null };
 
 	componentDidMount() {
 		store.on( 'change', this.onChange );
-		this.props.devices.get();
 		fetchSettings();
-	},
+	}
 
 	componentWillUnmount() {
 		store.off( 'change', this.onChange );
-	},
+	}
 
-	onChange() {
+	onChange = () => {
 		const state = store.getStateFor( 'other' );
 
 		if ( state.error ) {
-			this.props.errorNotice( this.translate( 'There was a problem saving your changes. Please, try again.' ) );
+			this.props.errorNotice( this.props.translate( 'There was a problem saving your changes. Please, try again.' ) );
```

Comment:
😄 It was just a note on how to avoid the shadow error in the future, since I found that some folks don't always know about the destructure-renaming behavior. 

I understand we want to move these action calls elsewhere.